src/PaletteCell.css

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2023 Inclusive Design Research Centre, OCAD University
+ * All rights reserved.
+ *
+ * Licensed under the New BSD license. You may not use this file except in
+ * compliance with this License.
+ *
+ * You may obtain a copy of the License at
+ * https://github.com/inclusive-design/adaptive-palette/blob/main/LICENSE
+ */
+
+.paletteCell {
```

Comment:
Using scss, line 12-28 can turn into a scoped nested structure:

```
.paletteCell {
  ...

  &:hover, &:focus {
    ...
  }

  & > svg {
    ...
  }
}
```

Reply:
Thanks for the suggestion.  I know little about scss nor how it works.  Time to learn :-)